lib/Db/MessageMapper.php

```diff
@@ -146,7 +147,7 @@ public function insertBulk(Message ...$messages): void {
 
 					$qb2->setParameter('message_id', $messageId, IQueryBuilder::PARAM_INT);
 					$qb2->setParameter('type', $type, IQueryBuilder::PARAM_INT);
-					$qb2->setParameter('label', $recipient->getLabel(), IQueryBuilder::PARAM_STR);
+					$qb2->setParameter('label', mb_substr($recipient->getLabel(), 0, 255, 'UTF-8'), IQueryBuilder::PARAM_STR);
```

Comment:
Should be enough without specifying the charset, see https://github.com/nextcloud/server/pull/19634